mdr/mdr.py

```diff
@@ -57,6 +61,19 @@ def fit(self, features, classes):
         None
 
         """
+        self.class_ratio = float(sum(classes == 0))/(classes.size) #only applies to binary classification 
```

Comment:
Rename `class_ratio` to `class_fraction`.

Use `sorted(np.unique(classes))[0]` in place of the hard-coded `0`.